docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst

```diff
@@ -0,0 +1,41 @@
+Integration of logs into Apache Kafka topic
+============================================
+
+You can send logs from your Aiven services into a specified Apache Kafka topic. The setup can be done through `Aiven console <https://console.aiven.io>`_.
+
+.. note::
+
+    Note, the integration can be used for both Aiven for Apache Kafka, as well as external Kafka clusters registered in the project's service integrations page. Read more on :doc:`how to manage Aiven internal and external integrations </docs/tools/cli/service/integration>`.
+
+In this example we'll show how to send the logs from an Aiven for PostgreSQL service to a topic in your Aiven for Apache
+Kafka service. For this example you need:
+
+-  a running Aiven for PostgreSQL service (we'll refer to it as *a source*)
+-  a running Aiven for Apache Kafka service (we'll refer as *a destination*)
+
+Setup Apache Kafka to receive the logs
```

Comment:
Set up is two words when you are setting something up. If you are talking about your shiny new setup (a noun) it's one word.